spec/requests/project_transfers_api_spec.rb

```diff
@@ -154,5 +154,56 @@
         let(:resource_id) {'notfoundid'}
       end
     end
+
+    describe 'Reject project transfer' do
+      let(:url) { "/api/v1/project_transfers/#{resource_id}/reject" }
+      let(:resource) { FactoryGirl.create(:project_transfer,
+                                          :with_to_users,
+                                          status: status,
+                                          to_user: current_user) }
+      let(:status) { :pending }
+      let(:payload) {{}}
+
+      describe 'PUT' do
+        subject { put(url, payload.to_json, headers) }
+        let(:called_action) { 'PUT' }
+
+        it_behaves_like 'an authenticated resource'
+        it_behaves_like 'an identified resource' do
+          let(:resource_id) {'notfoundid'}
+        end
+        it_behaves_like 'a software_agent accessible resource'
+        it_behaves_like 'an updatable resource' do
+          it 'sets project transfer status comment' do
+            is_expected.to eq(200)
+            expect(resource.reload).to be_truthy
+            expect(resource).to be_rejected
+          end
+        end
+        context 'when current user is not to_user' do
+          let(:resource) { FactoryGirl.create(:project_transfer,
+                                              :with_to_users,
+                                              status: status) }
+          it_behaves_like 'an authorized resource'
+        end
+        context 'with status_comment' do
+          let(:payload) {{
+            status_comment: Faker::Hacker.say_something_smart
+          }}
+          it_behaves_like 'an updatable resource' do
+            it 'sets project transfer status comment' do
+              is_expected.to eq(200)
+              expect(resource.reload).to be_truthy
+              expect(resource.status_comment).to eq(payload[:status_comment])
+            end
+          end
+        end
+        context 'where project_transfer status is rejected' do
```

Comment:
Change context description from "status is rejected" to "status is not processing".